doc/quicktask.txt

```diff
@@ -259,7 +290,15 @@ To open a snip, place your cursor anywhere on the snip's ID line and press
 <CR>. This mapping is customizable, see |quicktask-adv-mappings|.
 
 ===============================================================================
-7. Options                                      *quicktask-options*
+8. Options                                      *quicktask-options*
+
+g:quicktask_default_filename                    *g:quicktask_default_filename*
+    The name of the file to use by default to look for in parent directories.
```

Comment:
I think you can remove the "default" in the first sentence, because it is in the second sentence.

"The name of the file to open when no filename is given. The current directory and all parent directories are searched. Unless changed, this value is defaulted to `todo.txt`."

Or something like that?

Reply:
Alright. I replaced exactly by what you suggested!